R/repositories.R

```diff
@@ -68,11 +68,30 @@ BINARY_BASE_URL <- "https://bioconductor.org/packages/%s/container-binaries/%s"
     repos
 }
 
+.repositories_ci_mirror_envopt <-
+    function()
+{
+    .env_opt_lgl(
+        "BIOCMANAGER_USE_CI_MIRROR", "BiocManager.use_ci_mirror", TRUE
+    ) && as.logical(Sys.getenv("CI"))
+}
+
+.BIOCONDUCTOR_CI_MIRROR <- "https://ci-mirror.bioconductor.org"
+
+.repositories_bioc_mirror <- function() {
+    if (
+        .repositories_ci_mirror_envopt() && .url_exists(.BIOCONDUCTOR_CI_MIRROR)
```

Comment:
Is it good enough to know that mirror exists, or rather that the repository (e.g., packages/3.18/bioc) exists?

Reply:
I think it is good enough to know that the mirror exists. We can run health checks on it via [`BiocMirrors`](https://github.com/Bioconductor/BiocMirrors)